### ISSUE TITLE
kind v0.14.0 can't create a cluster on ppc64le

### ISSUE THREAD

**Issue by snehakpersistent** [kubernetes-sigs/kind#2797]
**What happened**:
During creation of cluster (kind create cluster) exception occured:
```
I0608 05:03:05.205238     179 round_trippers.go:553] GET https://test-cluster-control-plane:6443/healthz?timeout=10s  in 0 milliseconds
I0608 05:03:05.705016     179 round_trippers.go:553] GET https://test-cluster-control-plane:6443/healthz?timeout=10s  in 0 milliseconds
[kubelet-check] It seems like the kubelet isn't running or healthy.
[kubelet-check] The HTTP call equal to 'curl -sSL http://localhost:10248/healthz' failed with error: Get "http://localhost:10248/healthz": dial tcp [::1]:10248: connect: connection refused.

Unfortunately, an error has occurred:
        timed out waiting for the condition

This error is likely caused by:
        - The kubelet is not running
        - The kubelet is unhealthy due to a misconfiguration of the node in some way (required cgroups disabled)

If you are on a systemd-powered system, you can try to troubleshoot the error with the following commands:
        - 'systemctl status kubelet'
        - 'journalctl -xeu kubelet'

Additionally, a control plane component may have crashed or exited when started by the container runtime.
To troubleshoot, list all containers using your preferred container runtimes CLI.
Here is one example how you may list all running Kubernetes containers by using crictl:
        - 'crictl --runtime-endpoint unix:///run/containerd/containerd.sock ps -a | grep kube | grep -v pause'
        Once you have found the failing container, you can inspect its logs with:
        - 'crictl --runtime-endpoint unix:///run/containerd/containerd.sock logs CONTAINERID'
couldn't initialize a Kubernetes cluster
```

**What you expected to happen**:
Cluster should be created successfully 

**How to reproduce it (as minimally and precisely as possible)**:
`kind create cluster --image quay.io/mayurwaghmode111/node-ppc64le --name test-cluster`

**Anything else we need to know?**:
On version v0.12.0 cluster is created successfully on Power. Also above issue is similar to #2769 

**Environment:**

- kind version: (use `kind version`): v0.14.0
- Kubernetes version: (use `kubectl version`): v1.24.1
- Docker version: (use `docker info`): 20.10.0
- OS (e.g. from `/etc/os-release`): Centos 8(linux/ppc64le)

**Comment by BenTheElder** [comment]
how was this image built?

> Docker version: (use docker info): v1.24.1

that's not a valid docker version, can you please share the full `docker info` 

(@aojea we should probably change the template, to give full docker/podman info)

**Comment by BenTheElder** [comment]
My guess: this image is probably outdated and incompatible with kind v0.14.0, whatever this image is, it's not something this project produced or supported.

There are details on building your own node images in the user guide in the docs.

**Comment by snehakpersistent** [comment]
@BenTheElder sorry for the typo. Updated correct Docker version. Currently node image is not officially built on Power. We have built it locally and using it.

**Comment by BenTheElder** [comment]
can you share:
- `kind export logs` output (run `kind create cluster` with `--retain` flag to prevent cleanup on failure, then after exporting logs you can `kind delete cluster`)
- how precisely this node image was built, including the kind version used, the kubernetes version, and so on

**Comment by snehakpersistent** [comment]
[kind-logs.zip](https://github.com/kubernetes-sigs/kind/files/8858914/kind-logs.zip)
@BenTheElder attaching logs for your reference. Looks like its related to some cgroup config.

**Comment by aojea** [comment]
> Jun 08 06:23:53 test-cluster-control-plane kubelet[2110]: Error: failed to run Kubelet: invalid configuration: cgroup ["kubelet"] has some missing paths: /sys/fs/cgroup/systemd/kubelet.slice, /sys/fs/cgroup/cpu,cpuacct/kubelet.slice, /sys/fs/cgroup/cpu,cpuacct/kubelet.slice, /sys/fs/cgroup/hugetlb/kubelet.slice, /sys/fs/cgroup/cpuset/kubelet.slice, /sys/fs/cgroup/pids/kubelet.slice, /sys/fs/cgroup/memory/kubelet.slice

is not the problem that you've fixed recently ben?

**Comment by BenTheElder** [comment]
still waiting for details on the image build, this sounds like an image built with a previous version.

**Comment by snehakpersistent** [comment]
Thanks @BenTheElder for pointing that! We built fresh node image and the cluster is getting created successfully on Power. It would be really great if we could have official support for ppc64le node image. We can contribute the required changes. We have worked on cross-building of images using GitHub Actions and Qemu in past. 
One such use case is mentioned in this blog by my team: https://medium.com/@mayurwaghmode/building-multi-arch-container-images-with-github-actions-and-buildah-dbb383474824
Let me know your views on this.
